http/org_service.go

```diff
@@ -98,38 +98,48 @@ func (o orgsResponse) ToPlatform() []*platform.Organization {
 	return orgs
 }
 
-func newOrgsResponse(orgs []*platform.Organization) *orgsResponse {
+func newOrgsResponse(ctx context.Context, orgs []*platform.Organization, labelService platform.LabelService) *orgsResponse {
```

Comment:
I kind of hate passing context/labelservice to these functions, because pulling label information seems outside of the concerns of the function. I'll think about it today.